rest/model/project.go

```diff
@@ -535,87 +580,123 @@ func (t *APIRepositoryErrorDetails) BuildFromService(h model.RepositoryErrorDeta
+	// Whether or not to default to using repo settings.
+	UseRepoSettings *bool `json:"use_repo_settings"`
+	// Identifier of the attached repo ref. Cannot be modified by users.
+	RepoRefId *string `json:"repo_ref_id"`
+	// Options for commit queue.
+	CommitQueue APICommitQueueParams `json:"commit_queue"`
+	// Options for task sync.
+	TaskSync APITaskSyncOptions `json:"task_sync"`
+	// Options for task annotations.
 	TaskAnnotationSettings APITaskAnnotationSettings `json:"task_annotation_settings"`
-	BuildBaronSettings     APIBuildBaronSettings     `json:"build_baron_settings"`
-	PerfEnabled            *bool                     `json:"perf_enabled"`
-	Hidden                 *bool                     `json:"hidden"`
-	// Disable patching
-	PatchingDisabled      *bool                      `json:"patching_disabled"`
-	RepotrackerDisabled   *bool                      `json:"repotracker_disabled"`
-	RepotrackerError      *APIRepositoryErrorDetails `json:"repotracker_error"`
-	DispatchingDisabled   *bool                      `json:"dispatching_disabled"`
-	StepbackDisabled      *bool                      `json:"stepback_disabled"`
-	StepbackBisect        *bool                      `json:"stepback_bisect"`
-	VersionControlEnabled *bool                      `json:"version_control_enabled"`
-	DisabledStatsCache    *bool                      `json:"disabled_stats_cache"`
+	// Options for Build Baron.
+	BuildBaronSettings APIBuildBaronSettings `json:"build_baron_settings"`
+	// Enable the performance plugin.
+	PerfEnabled *bool `json:"perf_enabled"`
+	// Whether or not the project can be seen in the UI. Cannot be modified by
+	// users.
+	Hidden *bool `json:"hidden"`
+	// Disable patching.
+	PatchingDisabled *bool `json:"patching_disabled"`
+	// Disable the repotracker.
+	RepotrackerDisabled *bool `json:"repotracker_disabled"`
+	// Error from the repotracker, if any. Cannot be modified by users.
+	RepotrackerError *APIRepositoryErrorDetails `json:"repotracker_error"`
+	// Disable task dispatching.
+	DispatchingDisabled *bool `json:"dispatching_disabled"`
+	// Disable stepback.
+	StepbackDisabled *bool `json:"stepback_disabled"`
+	// Use bisect stepback instead of linear.
+	StepbackBisect *bool `json:"stepback_bisect"`
+	// Enable setting project aliases from version-controlled project configs.
+	VersionControlEnabled *bool `json:"version_control_enabled"`
+	// Disable stats caching.
+	DisabledStatsCache *bool `json:"disabled_stats_cache"`
 	// Usernames of project admins. Can be null for some projects (EVG-6598).
 	Admins []*string `json:"admins"`
-	// Usernames of project admins to remove
-	DeleteAdmins                []*string `json:"delete_admins,omitempty"`
-	GitTagAuthorizedUsers       []*string `json:"git_tag_authorized_users" bson:"git_tag_authorized_users"`
+	// Usernames of project admins to remove.
+	DeleteAdmins []*string `json:"delete_admins,omitempty"`
+	// Usernames authorized to submit git tag versions.
```

Comment:
Should we move the git tag disabled flag closer to this since they're so related?

Reply:
I dunno, the organization seems alright to me since it groups by the enabled/disabled flags rather than related features. Anyways, the fields are always listed in alphabetical order in the autogenerated docs so I don't think it matters to users.